server/src/app.module.ts

```diff
@@ -34,7 +35,7 @@ import { ExceptionFilter } from './filters/exception.filter';
       context: ({ req }) => ({ req }),
       driver: YogaDriver,
       autoSchemaFile: true,
-      include: [CoreModule],
+      include: [CoreV2Module],
```

Comment:
@gitstart-twenty We want to keep CoreModule working during the migration, so here you should keep `CoreModule` and create a new `GraphQLModule` configuration for `CoreV2Module` with a dedicated path.
Here an example with `metadata`:


```typescript
    GraphQLModule.forRoot<YogaDriverConfig>({
      context: ({ req }) => ({ req }),
      driver: YogaDriver,
      autoSchemaFile: true,
      include: [MetadataModule],
      resolvers: { JSON: GraphQLJSON },
      plugins: [],
      path: '/metadata',
    }),
```

Instead of metadata we can put `graphqlV2`